### DISCUSSION TITLE
Ripcord

### DISCUSSION THREAD

**Discussion by ManuKushwah** [catppuccin/catppuccin#2004] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

ripcord is a lightweight discord client 
here's the official site - https://cancel.fm/ripcord/

To customize the theme : 
view > preferences > style > customize (on right to 'theme') 

theme i'm currently using (can be used by copying it and clicking 'Paste from JSON' in the dialog box which appeared earlier) : 
{
    "alternate_base": "#00121e",
    "base": "#00131f",
    "button": "#001723",
    "chat_timestamp": "#ff007f",
    "disabled_button": "#292a2c",
    "disabled_icon": "#55aaff",
    "disabled_text": "#8d8b8b",
    "highlight": "#4d4c4c",
    "highlighted_text": "#55ff00",
    "icon": "#55aaff",
    "text": "#dad9d9",
    "unread_badge": "#d4d1d0",
    "unread_badge_text": "#2d2c27",
    "window": "#00141f"
}

![image1](https://user-images.githubusercontent.com/128598013/229993438-92f23c1a-547f-4316-8061-cffce394d54e.png)

how it looks : 
![image2](https://user-images.githubusercontent.com/128598013/229993397-d1d12c37-915d-4e23-9373-b21f39f99e4e.png)

### Any additional comments?

ripcord might not be that mainstream 
but its imo the best replacement to discord when you have a trash pc and also its highly customizable 
it gets the work done without compromising system resources , it also has tabs.
it even has voice chat!  i cannot imagine doing anything else while in vc in discord but with ripcord i can even game while in vc on my dual core ;) 

i can definitely work on a catppuccin theme 

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by vanillyn** [comment]
* since ripcord is a client that uses your user account to interact with the api on a non-discord service, usage of it can be detected and your discord account can be disabled/banned depending on how often you use it

**Reply by ManuKushwah** [reply]
i've been using it for like 1yr+ pretty much everyday now 
everything works just fine 
also isnt the point just to develope a theme for whoever might want to use it?

**Comment by AnubisNekhet** [comment]
Hasn't ripcord been discontinued for a long time?

**Reply by ManuKushwah** [reply]
they dont really push updates and the last release was on 2021 
but works for someone who wants to use it , everything is just the same
